### ISSUE TITLE
unable to automatically install Rmath

### ISSUE THREAD

**Issue by LWKJJONAK** [JuliaMath/RandomMatrices.jl#53]
I'm using Julia1.3.1 now and today I failed to install RandomMatrices.This is my status
`[c7e460c6] ArgParse v1.1.0

  [6e4b80f9] BenchmarkTools v0.5.0
  [50ba71b6] BitBasis v0.6.1
  [c5f51814] CUDAdrv v6.0.0
  [be33ccc6] CUDAnative v2.10.2
  [b48ca7a8] CuYao v0.2.0
  [5789e2e9] FileIO v1.2.2
  [587475ba] Flux v0.10.1
  [7073ff75] IJulia v1.21.1
  [033835bb] JLD2 v0.1.12
  [d05aeea4] LuxurySparse v0.5.4
  [23992714] MAT v0.7.0
  [15e1cf62] NPZ v0.4.0
  [e0fc9d43] PkgMirrors v1.3.0
  [91a5bcdd] Plots v0.29.4
  [2913bbd2] StatsBase v0.32.1
  [5872b779] Yao v0.6.1
  [7a06699c] YaoExtensions v0.2.0
  [e88e6eb3] Zygote v0.4.8
  [37e2e46d] LinearAlgebra
  [de0858da] Printf
  [9a3f8284] Random
  [10745b16] Statistics`

and this is the error report

`(v1.3) pkg> add RandomMatrices

 Resolving package versions...
 Installed Rmath_jll ─────────── v0.2.2+0
 Installed QuadGK ────────────── v2.3.1
 Installed StatsFuns ─────────── v0.9.4
 Installed OpenBLAS_jll ──────── v0.3.7+6
 Installed FastGaussQuadrature ─ v0.4.2
 Installed Combinatorics ─────── v1.0.0
 Installed Arpack_jll ────────── v3.5.0+2
 Installed Arpack ────────────── v0.4.0
 Installed Rmath ─────────────── v0.6.1
 Installed PDMats ────────────── v0.9.11
 Installed RandomMatrices ────── v0.5.0
 Installed Polynomials ───────── v0.6.1
 Installed Distributions ─────── v0.22.5
 Installed GSL ───────────────── v0.6.0
ERROR: Unable to automatically install 'Rmath' from '/home/Jake/.julia/packages/Rmath_jll/dkU4a/Artifacts.toml'`
Anyone met this before?

**Comment by dlfivefifty** [comment]
I don't think this is caused by RandomMatrices.jl. Perhaps asking on the Julia slack or discourse board someone will be able to answer?

Sometimes wiping the `~/.julia` folder helps

**Comment by LWKJJONAK** [comment]
> I don't think this is caused by RandomMatrices.jl. Perhaps asking on the Julia slack or discourse board someone will be able to answer?
> 
> Sometimes wiping the `~/.julia` folder helps

Thanks, I'll have a try ^_^

**Comment by LWKJJONAK** [comment]
> I don't think this is caused by RandomMatrices.jl. Perhaps asking on the Julia slack or discourse board someone will be able to answer?
> 
> Sometimes wiping the `~/.julia` folder helps

btw I'd like to ask that is RandomMatrices compatible with the current Julia1.3.1 or higher versions?
